### PR TITLE
List Particl (PART)

### PR DESCRIPTION
**Pull request by nikchis**
- Official project URL: https://particl.io
- Official block explorer URL: https://explorer.particl.io

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/Particl.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import bisq.asset.Base58BitcoinAddressValidator;
+import bisq.asset.Coin;
+import bisq.asset.NetworkParametersAdapter;
+import bisq.asset.AddressValidationResult;
+
+
+public class Particl extends Coin {
+    public Particl() {
+        super("Particl", "PART", new ParticlMainNetAddressValidator(), Network.MAINNET);
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Particl.java]: `Network.MAINNET` is a default value, so remove that param.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: assets/src/main/java/bisq/asset/coins/Particl.java (hunk 1 above)
Comment (nikchis) [reply assets/src/main/java/bisq/asset/coins/Particl.java]: Done.

**Comment by blabno** [comment]
Squash commits into a single one.

**Comment by nikchis** [comment]
> Squash commits into a single one.

Squashed. :ok_hand:

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**